userspace/sysdig/sysdig.cpp

```diff
@@ -186,29 +186,29 @@ static void usage()
 "                    create a ring buffer of events.\n"
 " -h, --help         Print this page\n"
 " -H <pluginname>[:<initconfig>], --plugin <pluginname>[:<initconfig>]\n"
-"                    marks a plugin as selected, storing its init config if present.\n"
+"                    registers a plugin, using the passed init config if present.\n"
 "                    The format of initconf is controlled by the plugin, refer to each\n"
 "                    plugin's documentation to learn about it.\n"
-"                    A path can also be passed as pluginname.\n"
-" -I <inputname>[:<openparams>], --input <inputname>[:<openparams>]\n"
-"                    registers a previously selected plugin.\n"
-"                    If the plugin is a source plugin, capture events using it, passing to the \n"
-"                    plugin the openparams string as open parameters.\n"
+"                    A path can also be used as pluginname.\n"
+" -I <pluginname>[:<openparams>], --input <pluginname>[:<openparams>]\n"
+"                    set a previously registered plugin as input plugin,\n"
+"                    capturing events using it and passing the \n"
+"                    openparams string as open parameters.\n"
 "                    Only a single source plugin can be registered.\n"
-"                    If no plugins were selected, registers any found plugin in the directories\n"
+"                    If no plugins were selected, any found plugin in the directories\n"
```

Comment:
```suggestion
"                    If no plugins were registered, any found plugin in the directories\n"
```
according to the previous sentence.

@FedeDP

Reply:
You're right, nice catch :)